glam/api/management/commands/import_desktop_aggs.py

```diff
@@ -58,6 +58,7 @@ def handle(self, bucket, *args, **options):
 
         for blob in blobs:
             # Create temp table for data.
+            print(blob)
```

Comment:
Do we need to print this for debugging? Not sure what it looks like but it might be a big thing to print?